packages/cli/src/runners/errors/task-runner-disconnected-error.ts

```diff
@@ -1,7 +1,34 @@
+import type { TaskRunner } from '@n8n/task-runner';
 import { ApplicationError } from 'n8n-workflow';
 
 export class TaskRunnerDisconnectedError extends ApplicationError {
-	constructor(runnerId: string) {
-		super(`Task runner (${runnerId}) disconnected`);
+	public description: string;
+
+	constructor(
+		public readonly runnerId: TaskRunner['id'],
+		isCloudDeployment: boolean,
+	) {
+		super('Node execution crashed', { level: 'error' });
```

Comment:
Maybe we should reserve the word `crash` for instance crashes, so it's less alarming/confusing for users. Also `error` level should be the default iirc.

```suggestion
		super('Node execution failed');
```